app/institutions/dashboard/template.hbs

```diff
@@ -7,20 +7,20 @@
             <Institutions::Dashboard::-Components::InstitutionalUsersList @model={{this.model}} />
         </div>
         <div local-class='panel-wrapper'>
-            <Panel as |panel|>
-                <panel.heading @title='SSO Users Connected' />
+            <Panel local-class='sso-users-connected' as |panel|>
+                <panel.heading @title='{{t 'institutions.dashboard.users_connected_panel'}}' />
                 <panel.body>
-                    <p></p>
+                    {{this.institution.statSummary.ssoUsersConnected}}
```

Comment:
Please add a loading indicator as the institution won't be immediately available.
```suggestion
                    {{#if this.institution}}
                        {{this.institution.statSummary.ssoUsersConnected}}
                    {{else}}
                        <LoadingIndicator @dark={{true}} />
                    {{/if}}
```